sys/net/credman/credman.c

```diff
@@ -90,6 +105,247 @@ int credman_add(const credman_credential_t *credential)
     return ret;
 }
 
+#if IS_USED(MODULE_CREDMAN_LOAD)
+int credman_load_public_key(const uint8_t *buf, size_t buf_len, ecdsa_public_key_t *out)
+{
+    assert(buf);
+    assert(out);
+
+    int obj_count = der_object_count((uint8_t *)buf, buf_len);
+    if (obj_count <= 0) {
+        DEBUG("credman: could not calculate the number of elements within the key\n");
+        return CREDMAN_INVALID;
+    }
+
+    asn1_tree objects[obj_count];
```

Comment:
Is there an alternative to using VLA here?

Reply:
I defined now a compile configuration for this, and a debug message when not enough objects are allocated.